### PR TITLE
Fix [Issue 25]: elasticsearch download link

### PR DESCRIPTION
**Pull request by rahuldkjain**
add a download link for all versions to index.html

### CODE REVIEW

**Comment by rahuldkjain** [comment]
@pyrech @damienalexandre Please review the PR

**Review by damienalexandre — CHANGES_REQUESTED** [review]
Ideally, for ES 7, it should be a link to https://www.elastic.co/downloads/elasticsearch as it's automatically the latest version (so it's quicker).

But for all the other versions your link is good :+1: 

Could you change this please? You have to use the classes "es1 es2" to display for Elastic 1 and 2, etc.

**Comment by rahuldkjain** [comment]
@damienalexandre made the required changes, please review again.

File: index.html [hunk 1]
```diff
@@ -161,6 +161,9 @@ <h2 class="ui dividing header">Links</h2>
           <li class="es2"><a href="https://www.elastic.co/guide/en/elasticsearch/guide/2.x/index.html">Elasticsearch: The Definitive Guide</a>, official book;</li>
           <li class="es5 es6"><a href="https://www.elastic.co/guide/en/elasticsearch/guide/master/index.html">Elasticsearch: The Definitive Guide</a>, official book (not up to date);</li>
 
+          <li class="es1 es2 es3 es4 es5 es6"><a href="https://www.elastic.co/downloads/past-releases/#elasticsearch">Download Elasticsearch</a>, official download page;</li>
```
Comment (damienalexandre) [inline index.html]: es3 and es4 do not exists :grimacing: (there is no such version)

**Review by damienalexandre — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (rahuldkjain) [reply index.html]: Sorry for that

Comment (rahuldkjain) [reply index.html]: @damienalexandre Review again

**Comment by damienalexandre** [comment]
Thank you!

**Pull request merged**